portal-frontend/src/app/features/notifications/notification-details/proposal-details/proposal-details.component.html

```diff
@@ -13,7 +13,7 @@
 
   <div class="subheading2 grid-1">Total area of the SRW</div>
   <div class="grid-double">
-    {{ _notificationSubmission.totalArea }}
+    {{ _notificationSubmission.totalArea }} <ng-container *ngIf="_notificationSubmission.totalArea">ha</ng-container>
```

Comment:
nitpick: Use a span just to be consistent with other places we do this, not a big deal since the end functionality works basically the same.

Reply:
I suppose for consistency this makes sense, but we end up with a pointless span then.

Reply:
And that's why its a nitpick, which one is the lesser of two evils? 🙈